daprdocs/content/en/reference/api/metadata_api.md

```diff
@@ -0,0 +1,191 @@
+---
+type: docs
+title: "Metadata API reference"
+linkTitle: "Metadata API"
+description: "Detailed documentation on the Metadata API"
+weight: 800
+---
+
+Dapr has a metadata API that returns information about the sidecar allowing runtime discoverability. The metadata endpoint returns among other things, a list of the components loaded and the activated actors (if present).
+
+The Dapr metadata API also allows you to store additional information in they format of key-value pairs. 
+
+Note: The Dapr metatada endpoint is for instance being used by the dapr cli when running dapr in standalone mode to store the PID of the process hosting the sidecar and the command used to run the application.
+
+## Get the Dapr sidecar information
+
+Gets the Dapr sidecar information provided by the Metadata Endpoint.
+
+### HTTP Request
+
+```http
+GET http://localhost:<daprPort>/v1.0/metadata
+```
+
+### URL Parameters
+
+Parameter | Description
+--------- | -----------
+daprPort  | The Dapr port.
+
+### HTTP Response Codes
+
+Code | Description
+---- | -----------
+200  | Metadata information returned
+500  | Dapr could not return the metadata information
+
+### HTTP Response Body
+
+**Metadata API Response Object**
+
+Name                   | Type                                                                  | Description
+----                   | ----                                                                  | -----------
+id                     | string                                                                | Application ID
+actors                 | [Metadata API Response Registered Actor](#metadataapiresponseactor)[] | A json encoded array of Registered Actors metadata.
+extended.attributeName | string                                                                | List of custom attributes as key(attributeName)-value pairs.
```

Comment:
... key-value pairs, where key is the attribute name.

Reply:
Thanks for the review. I have just updated the PR to address your comments.